yt_dlp/extractor/twitter.py

```diff
@@ -642,31 +865,30 @@ def get_binding_value(k):
 
             card_name = card['name'].split(':')[-1]
             if card_name == 'player':
-                return {
+                return [{
```

Comment:
I'd change these to `yield` instead, but this is fine too

Reply:
`yield`ing seems cleaner. I will change it.